packages/core/src/rum/instrumentation/DdRumUserInteractionTracking.tsx

```diff
@@ -37,27 +62,33 @@ export class DdRumUserInteractionTracking {
             return;
         }
         DdRumUserInteractionTracking.eventsInterceptor = new DdEventsInterceptor();
+
         const original = React.createElement;
         React.createElement = (
-            element: any,
-            props: any,
-            ...children: any
+            ...args: Parameters<typeof React.createElement>
         ): any => {
-            // check if we have an 'onPress' property and that this is really a function
-            if (props && typeof props.onPress === 'function') {
-                const originalOnPress = props.onPress;
-                props.onPress = (...args: any[]) => {
-                    DdRumUserInteractionTracking.eventsInterceptor.interceptOnPress(
-                        ...args
-                    );
-                    return originalOnPress(...args);
-                };
-                // we store the original onPress prop so we can keep memoization working
-                props.__DATADOG_INTERNAL_ORIGINAL_ON_PRESS__ = originalOnPress;
-            }
-            return original(element, props, ...children);
+            return this.patchCreateElementFunction(original, args);
         };
 
+        try {
+            // We have to use inline require here because older React versions (below 17) don't have jsx-runtime
+            // eslint-disable-next-line global-require, @typescript-eslint/no-var-requires
+            const jsxRuntime = require('react/jsx-runtime');
```

Comment:
if module is not found, then `require` will throw an exception? that is why we have empty `catch` block, because it is too noisy and will always be hit on pre-React 17?

Reply:
yes if the module is not found an error will be thrown